flink-formats/flink-protobuf/src/test/proto/test_map.proto

```diff
@@ -22,16 +22,14 @@ option java_package = "org.apache.flink.formats.protobuf.testproto";
 option java_multiple_files = true;
 
 message MapTest {
-    optional int32 a = 1;
-    map<string, string> map1 = 2;
-    map<string, InnerMessageTest> map2 = 3;
-    map<string, bytes> map3 = 4;
+  optional int32 a = 1;
```

Comment:
Reformat to follow the style guide: https://developers.google.com/protocol-buffers/docs/style